scripts/include-config.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Config, Asset Types and Custom Attributes are included in app bundle
+ * for better performance, since these resources are blocking in several ways,
+ * impacting Home and Search pages.
+ * Categories are not included in bundle as there can be many of these.
+ */
+
+const { initStelaceSdk } = require('./sdk')
+
+const dotenv = require('dotenv')
+const { isEmpty } = require('lodash')
+const path = require('path')
+const fs = require('fs')
+const util = require('util')
+const writeFile = util.promisify(fs.writeFile)
+
+const chalk = require('chalk')
+const log = console.log
+const success = str => log(`\n${chalk.green(str)}\n`)
+const warn = (err, msg) => {
+  if (msg) log(chalk.yellow(msg))
+  if (err) log(err)
+}
+
+dotenv.config({
+  path: process.env.NODE_ENV === 'production' ? '.env.production' : '.env.development'
+})
+
+const apiBaseURL = process.env.STELACE_API_URL
+const apiKey = process.env.STELACE_PUBLISHABLE_API_KEY
+
+const stelace = initStelaceSdk({ apiBaseURL, apiKey })
+// Don’t delay build for too long if network fails
+stelace.setTimeout(5000)
+
+run()
+  .then(() => success('API Config, Asset Types and Custom Attributes copied for build'))
```

Comment:
Exactly. It’s not required to load the app, but just some performance improvement.

Though a catch block should actually ensure we have some empty files at least, to avoid errors there https://github.com/stelace/marketplace-demo/blob/1c6741bffaa927695e6629ad49f3b3ec98db0582/src/store/common/state.js#L4-L6 (and `*LastFetchedDate` set to null in this store file then).

Good catch.